src/pmlogger/pmlogger_daily_report.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
I think this file has been accidentally added?  (brought back accidentally?, it previously was deleted I think)